lib/train/platforms/detect/specifications/os.rb

```diff
@@ -76,6 +76,13 @@ def self.load
             true
           end
         end
+      plat.name("kali").title("Kali Linux").in_family("debian")
+        .detect do
+          if linux_os_release && linux_os_release["ID"].eql?("kali")
```

Comment:
`eql?` is _never_ the right comparator to call and can't succeed in the wild. Please stick with `==`.

Reply:
Updated in latest PR.